src/main/kotlin/lotto/domain/LottoNumber.kt

```diff
@@ -0,0 +1,21 @@
+package lotto.domain
+
+class LottoNumber(val number: Int) {
+    companion object {
+        private const val MIN_NUMBER = 1
+        private const val MAX_NUMBER = 46
+
+        val CACHED_NUMBER = List(MAX_NUMBER) { num ->
```

Comment:
```suggestion
        val CACHED_NUMBER = (MIN_NUMBER .. MAX_NUMBER).map(::LottoNumber)
```
이런식으로 작성하면 -1 같은 추가 연산이 필요 없을 것 같네요